File: java/MaratonaJava/javinha/src/main/java/com/maratonajava/estrutura_sequencial/exerc02.java
package com.maratonajava.estrutura_sequencial;

import java.util.Scanner;

public class exerc02 {
    //Faça um Programa que peça um número e então mostre a mensagem O número informado foi [número]. 
    public static void main(String[] args){
        try(Scanner scan = new Scanner(System.in)){
            int num1;

            System.out.println("Digite um número.");
            num1 = scan.nextInt();

            System.out.println("O Número digitado foi: "+num1);
        }

    }
    
}

File: C/Exercicios/Estrutura_Sequencial/exerc02.c
#include <stdio.h>

#define texto "Programa que peça um número e então mostre a mensagem O número informado foi [número]\n\n"

int main(){
    //Faça um Programa que peça um número e então mostre a mensagem O número informado foi [número]
    printf("%s",texto);

    int num;

    printf("Digite um número.\n");
    scanf("%d", &num);

    printf("O número informado foi: %d. \n ", num);

}
